pom.xml

```diff
@@ -72,14 +72,6 @@
       <scope>system</scope>
       <systemPath>${project.basedir}/lib/PreciousStones 10.6.1.jar</systemPath>
     </dependency>
-    <!-- PK RPG -->
-    <dependency>
-      <groupId>com.projectkorra</groupId>
-      <artifactId>rpg</artifactId>
-      <version>LATEST</version>
-      <scope>system</scope>
-      <systemPath>${project.basedir}/lib/ProjectKorraRPG.jar</systemPath>
-    </dependency>
     <!-- PK ITEMS -->
```

Comment:
After your changes to the help command, PK Items is no longer required as build depenency for ProjectKorra Core. Might as well remove Spirits while you are at it, just remove the following code in WhoCommand at line 312-323.

```java
if (GeneralMethods.hasSpirits()) {
	final SpiritPlayer sPlayer = SpiritPlayer.getSpiritPlayer(player_);
	if (subelement.equals(SpiritElement.DARK) && sPlayer.isLightSpirit()) {
		sender.sendMessage(subelement.getColor() + "    Is " + sPlayer.getSpirit().getName() + element.getName());
	}
	if (subelement.equals(SpiritElement.LIGHT) && sPlayer.isDarkSpirit()) {
		sender.sendMessage(subelement.getColor() + "    Is " + sPlayer.getSpirit().getName() + element.getName());
	}
	if (sPlayer.isSpirit()) {
		continue;
	}
}
```